packages/manager/core/generator/app/templates/src/components/ButtonTooltip/ButtonTooltip.tsx

```diff
@@ -39,7 +39,7 @@ const ButtonTooltip: React.FC<ButtonTooltipProps> = (props) => {
   }, []);
 
   const handleTooltipToggle = (
-    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    event: React.MouseEvent<HTMLOsdsButtonElement, MouseEvent>,
```

Comment:
Is the type `HTMLOsdsButtonElement` available at a global level ? or did you forgot to import it ?

Reply:
It's available globally when you import OdsButton